spec/requests/citizens/check_answers_spec.rb

```diff
@@ -68,8 +76,34 @@
       expect(response).to redirect_to(flow_forward_path)
     end
 
-    it 'does not change the state' do
-      expect { subject }.not_to change { legal_aid_application.reload.state }
+    it 'changes the state' do
+      expect { subject }.to change { legal_aid_application.reload.state }
+    end
+
+    it 'sets the application state to analysing_bank_transactions' do
+      subject
+      expect(legal_aid_application.reload.state).to eq 'analysing_bank_transactions'
+      expect(legal_aid_application.completed_at).to be_within(1).of(Time.current)
+    end
+
+    it 'changes the provider step to start_merits_assessment' do
+      subject
+      expect(legal_aid_application.reload.provider_step).to eq('client_completed_means')
+    end
+
+    it 'records when the declartion accepted' do
```

Comment:
I think this should be `records when the declaration was accepted ` (not your error, it was wrong in the original declarations_spec!)